src/Exercise10/Exercise10SelfCheckoutTest.java

```diff
@@ -62,7 +62,7 @@ public void threeItemsDifferentPriceDifferentQuantities() {
 
     @Test
     public void taxOneItem() {
-        assertThat(exercise10SelfCheckout.taxCalculator(new Item("Coffee", 5, 1))).isEqualTo(0.275);
```

Comment:
It's a bit weird to be expecting half-pence values - these tests will fail as I've amended them but we should probably be choosing a rounding rule to avoid fractions of a penny.